src/Adapter/Form/ChoiceProvider/MerchandiseReturnStateChoiceProvider.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/**
+ * 2007-2020 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
```

Comment:
We have updated our license headers to remove the 2020 (so we dont need to update them every year), see https://devdocs.prestashop.com/1.7/development/coding-standards/

You dont need to fix this PR :) I will do it later with a tool, but update your IDE settings